install-pre-commit-hook.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Link pre-commit hook for a repo in the directory supplied as an argument, or the parent repo of this file.
+#
+# USAGES:
+# $ /path/to/install-pre-commit-hook.sh
+# $ /path/to/install-pre-commit-hook.sh wp-content/themes/my-theme
+
+set -e
+
+# Store the initial directory to return back to it later
+OLDPWD=`pwd`
+
+# Navigate to the script directory
+cd `dirname $0`
+
+# Get full path of the script directory
+BINDIR=`pwd`
+
+# Get back to the calling directory
+cd - &> /dev/null
+
+# If passed a destination, switch to it to get the full path and then .git/hooks directory
+if [ ! -z $1 ]; then
```

Comment:
If no arguments are passed, this will be a syntax error I believe. You could either use `[[ ... ]]` or you could quote the var like `"$1"`.